### DISCUSSION TITLE
希望大佬有空能在【下载视频】模块做个音频下载功能

### DISCUSSION THREAD

**Discussion by yorokami** [the1812/Bilibili-Evolved#3302] [功能建议 / Ideas]
个人有关注一些混音类的up，因为有些up只发视频，自己抽取视频的的音轨还是挺麻烦的。希望大佬有空能在【下载视频】模块做个音频下载功能。

**Comment by Geeyun-JY3** [comment]
好像可以啊：#2325
https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.0.0-tp9

**Reply by the1812** [reply]
对, 早就支持了
